packages/ckeditor5-engine/src/view/domconverter.ts

```diff
@@ -1593,7 +1593,7 @@ export default class DomConverter {
 
 		// If any of node ancestors has a name which is in `preElements` array, then currently processed
 		// view text node is (will be) in preformatted element. We should not change whitespaces then.
```

Comment:
This comments need updating (mentioning the `white-space` property).

Reply:
I've simplified the comment by just mentioning if it is preformatted, because `_isPreFormatted` already documents the scenarios in which a node is considered preformatted.
I did notice that the JSDoc on `_isPreFormatted` mentioned `<pre>` elements directly and not the `preElements` array, so I've corrected it there as well.